tests/Resources/ti.geolocation.addontest.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe.windowsBroken('Titanium.Geolocation', function () {
+	const isiOS14 =  (parseInt(Ti.Platform.version.split('.')[0]) >= 14);
```

Comment:
cc @vijaysingh-axway @jquick-axway @garymathews Seem to me like over time we tend to eventually write some code like this to guard tests (and I assume devs do too to guard codepaths). Any suggestions on an API to help not have to do this ugly parsing? Maybe just shrug and say use the `semver` npm package in your app for version comparisons?

Reply:
Can we expose API something like Ti.Utils.isIOSVersionOrGreater('14.0') inside SDK?  Internally in iOS we use https://github.com/appcelerator/titanium_mobile/blob/adf38f33dbaa7dee244d3ef89154dab97c705271/iphone/TitaniumKit/TitaniumKit/Sources/API/TiUtils.m#L215

Reply:
On Android, we already have an `API_LEVEL` property. This is typically used over Android OS version major/minor numbers. But there is no cross-platform equivalent of this.
https://docs.appcelerator.com/platform/latest/#!/api/Titanium.Platform.Android-property-API_LEVEL

I think exposing an OS `versionMajor` and `versionMinor` as numeric properties would work well. All platforms can support this, including Android. If we were to re-add Windows support in the future, then we would also need to add a `versionBuild` property.

Reply:
So a `Ti.Platform.versionMajor` and `Ti.Platform.versionMinor` represented as numbers (integers)?

Reply:
Yeah, just like that. That would make it look nice and simple like this...
```javascript
if (OS_IOS && (Ti.Platform.versionMajor >= 14)) {
	// Uses newest iOS feature.
}
```

Since we're pushing devs to use the `OS_IOS` and `OS_ANDROID` constants, do you think we should add new `OS_VERSION_MAJOR` and `OS_VERSION_MINOR` constants too? We can define them in our "ti.main.js" on app startup like this...
```javascript
//ti.main.js
global.OS_VERSION_MAJOR = Ti.Platform.versionMajor;
global.OS_VERSION_MINOR = Ti.Platform.versionMinor;
```

Then the app code can look this...
```javascript
if (OS_IOS && (OS_VERSION_MAJOR >= 14)) {
	// Uses newest iOS feature.
}
```